src/changes/changes.xml

```diff
@@ -5,6 +5,9 @@
         <author email="brunodepaulak AT yahoo DOT com DOT br">Bruno P. Kinoshita</author>
     </properties>
     <body>
+<action dev="ranamansoor" due-to="ranamansoor" type="add" issue="118">
```

Comment:
You can leave the changes to be added later @ranashahan , or use "kinow" in `dev`. Will amend once merged.